src/test/e2e/variable_test.go

```diff
@@ -39,7 +39,39 @@ func TestBundleVariables(t *testing.T) {
 	os.Setenv("UDS_CONFIG", filepath.Join("src/test/bundles/02-simple-vars", "uds-config.yaml"))
 
 	_, stderr := deploy(t, bundleTarballPath)
+	bundleVariablesTestChecks(t, stderr, bundleTarballPath)
+	remove(t, bundleTarballPath)
 
+	// Run same test checks but with package that isn't explicitly importing vars
+	bundleDir = "src/test/bundles/02-simple-vars/import-all"
+	bundleTarballPath = filepath.Join(bundleDir, fmt.Sprintf("uds-bundle-import-all-%s-0.0.1.tar.zst", e2e.Arch))
+	createLocal(t, bundleDir, e2e.Arch)
+	createRemoteInsecure(t, bundleDir, "localhost:888", e2e.Arch)
```

Comment:
Is `createRemoteInsecure` necessary? Do we test that creating remotely triggers the same error?

Reply:
removed in latest commit, good catch!